src/root/worker.js

```diff
@@ -0,0 +1,25 @@
+// Fake some DOM stuff for jQuery
+var document = self.document = {parentNode: null, nodeType: 9, toString: function() {return "FakeDocument"}};
+var window = self.window = self;
+var fakeElement = Object.create(document);
+fakeElement.nodeType = 1;
+fakeElement.toString=function() {return "FakeElement"};
+fakeElement.parentNode = fakeElement.firstChild = fakeElement.lastChild = fakeElement;
```

Comment:
This script runs in a web worker. It does not have access to a DOM per definition. As some dependency (docson) needs jQuery to load (although not accessing the DOM), we need to make jQuery loadable and this is done by this ugly snippet faking the necessary stuff in the DOM makeing jQuery able to start.
It could be replaced by something cleaner but I haven't found an easy one, yet. I think it is not so problematic, since this is clearly isolated in this script.